composer.lock

```diff
@@ -15248,5 +15298,5 @@
         "php": ">=8.0"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.3.0"
+    "plugin-api-version": "2.2.0"
```

Comment:
Doesn't matter much but the version of composer used for the current site images in 2.3.x.